### DISCUSSION TITLE
Feature Request: Navigating code on GitHub Enterprise (On-premise)

### DISCUSSION THREAD

**Discussion by hailey-shin** [community/community#11644] [Enterprise]
It would be great if GitHub Enterprise could be supported [Navigating code](https://docs.github.com/en/repositories/working-with-files/using-files/navigating-code-on-github).

GitHub is one of collaboration tools which requires space to strengthen the communication between the developers. 
In other words, developer will need to see/review the code written by fellow developers on GitHub.

We usually uses `clone` and identify on IDE, but sometimes as a user experience, it would be better to simply check it on the web without having to `clone` it. 

Please consider to provide this functionality in GitHub Enterprise and also in the On-premise environment.

**Comment by wreiske** [comment]
Any updates on this feature request would be greatly appreciated.
